### PR TITLE
Add libmkl_vml_def.so

### PR DESCRIPTION
**Pull request by CoderHam**
- Needed by PyTorch for CPU model execution

### CODE REVIEW

**Review by deadeyegoodwin — APPROVED** [review]

**Comment by CoderHam** [comment]
Rebased and ready to merge

**Review by deadeyegoodwin — APPROVED** [review]

**Review by deadeyegoodwin — COMMENTED** [review]
Don't you need to also explicitly link this library into libtriton_pytorch.so as we did with the other libraries?

**Comment by CoderHam** [comment]
> Don't you need to also explicitly link this library into libtriton_pytorch.so as we did with the other libraries?

Yes, I forgot to create the PR for it. I was already ready. Doing it now.

File: build.py [hunk 1]
```diff
@@ -559,6 +559,7 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap, backends):
 COPY --from=tritonserver_pytorch /opt/conda/lib/libmkl_intel_thread.so /opt/tritonserver/backends/pytorch/
 COPY --from=tritonserver_pytorch /opt/conda/lib/libmkl_def.so /opt/tritonserver/backends/pytorch/
 COPY --from=tritonserver_pytorch /opt/conda/lib/libmkl_avx2.so /opt/tritonserver/backends/pytorch/
+COPY --from=tritonserver_pytorch /opt/conda/lib//libmkl_vml_def.so /opt/tritonserver/backends/pytorch/
```
Comment (deadeyegoodwin) [inline build.py:562]: You have an extra backslash in there...

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**